perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/RegistrarManagerImpl.java

```diff
@@ -2213,11 +2211,23 @@ public Application getApplicationById(PerunSession sess, int appId) throws Regis
 	}
 
 	@Override
-	public List<Application> getOpenApplicationsForUser(PerunSession sess) {
+	public List<Application> getOpenApplicationsForUserInVo(User user, Vo vo) {
 
 		try {
-			List<Application> allApplications = jdbc.query(APP_SELECT + " where state in (?,?) order by a.id desc",
-				APP_MAPPER, AppState.VERIFIED.toString(), AppState.NEW.toString());
+			return jdbc.query(APP_SELECT + " where user_id=? and state in (?,?) and a.vo_id=? order by a.id desc",
+				APP_MAPPER, user.getId(), AppState.VERIFIED.toString(), AppState.NEW.toString(), vo.getId());
+		} catch (EmptyResultDataAccessException ex) {
+			return new ArrayList<>();
+		}
+
+	}
+
+	@Override
+	public List<Application> getOpenApplicationsForUserInVo(PerunSession sess, Vo vo) {
```

Comment:
Shouldn't the name be like this?
```suggestion
	public List<Application> getOpenApplicationsInVo(PerunSession sess, Vo vo) {
```

Reply:
No

Reply:
Ok